web/pdf_find_bar.js

```diff
@@ -25,8 +25,6 @@ const MATCHES_COUNT_LIMIT = 1000;
  * is done by PDFFindController.
  */
 class PDFFindBar {
-  #resizeObserver = new ResizeObserver(this.#resizeObserverCallback.bind(this));
```

Comment:
If this is being removed, then https://github.com/mozilla/pdf.js/blob/790470c1aadf4dc6c4e8ff00364b87ecea4d7bd7/web/app.js#L2132 is probably unnecessary.